kitsune/search/v2/documents.py

```diff
@@ -15,30 +19,41 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
-    updated = field.Date()
+    """ES Document for Knowledge Base.
+
+    Each document in ES is built on top of a parent KB article.
+    All the children are included in the same document.
+    """
 
+    updated = field.Date()
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
+    locale = field.Keyword()
+    category = field.Keyword()
+    recent_helpful_votes = field.Integer()
 
     # Document specific fields (locale aware)
+    url = SumoLocaleAwareKeywordField(store=True)
```

Comment:
What does `store=True` do here (and in all the other places)? I can't find it in the docs.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-store.html